workflow/cases/atm_free_forecast.yaml

```diff
@@ -0,0 +1,56 @@
+case:
+  places:
+    workflow_file: layout/free_forecast_gfs.yaml
+    #BASE_CPLIC: /scratch2/NCEPDEV/climate/climpara/S2S/IC
+
+  settings: 
+    SDATE: 2013-04-01t00:00:00
+    EDATE: 2013-04-01t00:00:00
+
+    cplflx: .false.
+    print_esmf: .true.
+    nems_temp: 'med_atm_ocn_ice'
+    nems_temp_cold: 'medcold_atm_ocn_ice'
+    mom6ic_prepared: .false.
+    KEEPDATA: NO
+    ic_source: cfsr
+
+  nsst:
+    NST_MODEL: 0
+
+  output_settings:
+    OCN_INTERVAL: 24
```

Comment:
What is OCN_INTERVAL? Is this an ATM variable? Otherwise this could be removed, along with the ocn_settings and ice_settings

Reply:
Agree. This should be removed.

Reply:
@JessicaMeixner-NOAA I've also changed the nems.configure templates to "atm" in most recent push.